docs/reference/commandline/diff.md

```diff
@@ -13,36 +13,53 @@ keywords: "list, changed, files, container"
      will be rejected.
 -->
 
-# diff
+## diff
```

Comment:
Weren't these generated from the metadata now?
Guess we need to do a full pass to remove the H1's everywhere